src/pages/settings/AboutPage/AboutPage.js

```diff
@@ -13,7 +13,7 @@ import withLocalize, {withLocalizePropTypes} from '../../../components/withLocal
 import withWindowDimensions, {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
 import MenuItem from '../../../components/MenuItem';
 import Logo from '../../../../assets/images/new-expensify.svg';
-import {version} from '../../../../package.json';
+import pkg from '../../../../package.json';
```

Comment:
Is this change necessary?

Reply:
Same as https://github.com/Expensify/App/pull/10286?notification_referrer_id=NT_kwDOAgaDu7M0MTQ2ODUxMDIyOjMzOTgxMzcx#discussion_r940681238